apis/projectcontour/v1alpha1/contourconfig.go

```diff
@@ -532,6 +544,21 @@ const (
 	AllClusterDNSFamily ClusterDNSFamilyType = "all"
 )
 
+// ServerHeaderTransformation defines the action to be applied to the Server header on the response path
+type ServerHeaderTransformationType int32
```

Comment:
Let's make this a string:
```suggestion
type ServerHeaderTransformationType string
```